### DISCUSSION TITLE
Is there an option to have Panels apply to all existing Windows and to New Windows?

### DISCUSSION THREAD

**Discussion by jwsmith601** [mbnuqw/sidebery#1732] [Q&A]
Am new to Sidebery so I appreciate any guidance.

1) Is there an option to have Panels apply to all existing Windows and to New Windows?

2) And perhaps a way to optionally Consolidate all Tabs in their respective Panels across all Windows?

**Comment by swiftness9** [comment]
I am pretty sure that 1 is by default, but 2 is non existent, but I am trying to add it in add it as show in https://github.com/mbnuqw/sidebery/discussions/1763#discussion-7061288

**Comment by DRYN07** [comment]
Is there any news about this by any chance? I also would like to use my whole setup in every window. I have read #1763 but did not find any newer post about this.

**Comment by tetraphoz** [comment]
I'm also really interested on the second workflow. Sidebery is so close to perfect for managing all my needs and not having "true" workspaces is kinda counterintuitive.
